<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFactoryEdit</class>
 <widget class="QDialog" name="DialogFactoryEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改厂家信息</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>2</y>
     <width>200</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>LOGO</string>
   </property>
   <widget class="QLabel" name="LbPic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>47</y>
      <width>180</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSetLogo">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>20</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确  定 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="EdRemark"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>备   注：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>简   称：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EdForShort"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="EdFactory"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>生产厂家：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EdWebSite"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>网   址：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
